--- ./drivers/video/fbdev/mxc/mxc_lcdif.c	2018-05-09 10:15:44.775983378 +0200
+++ ./drivers/video/fbdev/mxc/mxc_lcdif.c	2018-05-09 11:51:58.010627952 +0200
@@ -48,6 +48,12 @@
 	FB_SYNC_CLK_LAT_FALL,
 	FB_VMODE_NONINTERLACED,
 	0,},
+        {
+	/* 480x272 @ 60 Hz , pixel clk @ 9MHz */
+	"AMPIRE-AM480272MNTMQW-T00H", 60, 480, 272, 111111, 40, 2, 2, 2, 2, 10,
+	FB_SYNC_CLK_LAT_FALL,
+	FB_VMODE_NONINTERLACED,
+	0,},
 };
 static int lcdif_modedb_sz = ARRAY_SIZE(lcdif_modedb);
 
